projects/common/src/navigation/navigation.service.ts

```diff
@@ -75,6 +85,10 @@ export class NavigationService {
     return this.currentParamMap.getAll(parameterName);
   }
 
+  private getLeafRoute(activatedRoute: ActivatedRoute): ActivatedRoute {
```

Comment:
this is just an alternative impl to `getCurrentActivatedRoute` - don't need both.

Reply:
fixed